MediaBrowser.Model/Session/OrderMode.cs

```diff
@@ -0,0 +1,18 @@
+namespace MediaBrowser.Model.Session
+{
+    /// <summary>
+    /// Enum OrderMode.
+    /// </summary>
+    public enum OrderMode
```

Comment:
The enum would be called "PlaybackOrder" if we want to stay consistent with my example. With two values: Default and Shuffle